app.js

```diff
@@ -0,0 +1,107 @@
+//Guessing Game, Three questions
+//Quetions one
+//Do you own an umbrella
+//If yes, You are new to the Seattle area
+//If No, Good real Seattleites don't use umbrallas
+var counter = 0;
+var umbrella = "yes";
+var sunny = "yes";
+var good = "yes";
+var answer = "yes";
+
+
+while(counter < 4){
+var userInput = prompt("Do you own an umbrella?");
+if(umbrella === userInput){
+alert("You are new to the Seattle area");
+console.log("true");
+counter++;
+}else{
+alert("Good real Seattleites don't use umbrallas");
+console.log("false");
+}
+//Question two
+//Do you like Sunny weather?
+//If yes, Hope we have a long summer
+//If no, Winter is not that far away
+
+var weather = prompt("Do you like sunny weather?");
+if(sunny === weather){
```

Comment:
You forgot to do error checking on all your answers. For instance, I entered 'Yes' and it was not recognized.